check-runner.js

```
@@ -0,0 +1,767 @@
+  let payload;
+  try { payload = JSON.parse(base64UrlDecode(payloadB64).toString()); } catch { return null; }
+
+  const now = Math.floor(Date.now() / 1000);
+  if (typeof payload.exp === "number" && now > payload.exp) return null;
+
+  return payload;
+}
+
+/** @param {http.IncomingMessage} req */
+function isAuthorized(req) {
+  if (!JWT_SECRET) return true;
```

ESLint

[error] Expected { after 'if' condition.

(curly)